converter/ena/classes/__init__.py

```diff
@@ -0,0 +1,115 @@
+from converter.ena.classes.sra_common import (
+    AttributeType,
+    BasecallMatchEdge,
+    IdentifierType,
+    LinkType,
+    NameType,
+    ObjectType,
+    PipelineType,
+    PlatformType,
+    ProcessingType,
+    QualifiedNameType,
+    ReadSpecReadClass,
+    ReadSpecReadType,
+    RefObjectType,
+    ReferenceAssemblyType,
+    ReferenceSequenceType,
+    SequencingDirectivesType,
+    SequencingDirectivesTypeSampleDemuxDirective,
+    SpotDescriptorType,
+    Urltype,
+    XrefType,
+    Type454Model,
+    TypeAbiSolidModel,
+    TypeBgiseqmodel,
+    TypeCgmodel,
+    TypeCapillaryModel,
+    TypeDnbSeqModel,
+    TypeHelicosModel,
+    TypeIlluminaModel,
+    TypeIontorrentModel,
+    TypeOxfordNanoporeModel,
+    TypePacBioModel,
+)
+from converter.ena.classes.sra_experiment import (
+    Experiment,
+    ExperimentSet,
+    ExperimentSetType,
+    ExperimentType,
+    LocusLocusName,
```

Comment:
Is double `Locus` intentional?

Reply:
everything in the `converter.ena.classes` module is generated from ENA experiment and run XSDs.